changelog/unreleased/issue-2277

```diff
@@ -0,0 +1,15 @@
+Enhancement: add support for ppc64le
+
+It was added supported for ppc64le as part of Restic. 
```

Comment:
There's no need to mention Restic here, since this is already a natural context. Also, the grammar needs fixing.